extensions/keycloak-admin-client-common/runtime/src/main/java/io/quarkus/keycloak/admin/client/common/KeycloakAdminClientConfig.java

```diff
@@ -12,18 +12,21 @@
 @ConfigRoot(phase = ConfigPhase.RUN_TIME, name = "keycloak.admin-client")
 public class KeycloakAdminClientConfig {
 
-    /**
-     * Realm.
-     */
-    @ConfigItem(defaultValue = "master")
-    public String realm;
-
     /**
      * Keycloak server URL, for example, `https://host:port`.
+     * If this property is not set then the `Keycloak` Admin Client injection will fail - use
+     * {@linkplain org.keycloak.admin.client.KeycloakBuilder}
+     * to create it instead.
      */
     @ConfigItem
     public Optional<String> serverUrl;
 
+    /**
+     * Realm.
+     */
+    @ConfigItem(defaultValue = "master")
```

Comment:
Is it still called `master`, they haven't changed it to something else?

Reply:
@gsmet, right, in Keycloak the main realm is called this way, I guess they also have compatibility concerns at the API level since all sort of KC clients work via this realm